webapp/src/js/services/rules-engine.js

```diff
@@ -54,7 +54,7 @@ var nools = require('nools'),
         return doc && (
           doc.patient_id ||
           doc.place_id ||
-          (doc.fields && (doc.fields.patient_id || doc.fields.place_id))
+          (doc.fields && (doc.fields.patient_id || doc.fields.place_id || doc.fields.patient_uuid))
```

Comment:
Should this be a separate commit? Maybe it should even have its own issue?

Reply:
Yeah it's a good question. I like the idea of pulling it out into its own PR, because then when we merge it gets a separate commit message, so I'll do that (it'll have to be based on this PR, but that should be OK once we do the merge dance).